### ISSUE TITLE
TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by rchrdnsh** [AndreasFaust/gatsby-plugin-transitions#7]
not sure why I'm getting this, as your plugin was working just fine, but now when I try to implement the plugin I'm getting the following in-browser error:

<img width="1046" alt="Screenshot 2019-05-30 10 30 20" src="https://user-images.githubusercontent.com/7884263/58651420-222caf00-82c6-11e9-9a9e-4cd1cb9a2492.png">

all that I do to have this error show up is use the plugin, like so:

```jsx
<TransitionProvider location={location}>
  {children}
</TransitionProvider>
```

and when I turn off the plugin by replacing that code with this code:

```jsx
<Main>
  {children}
</Main>
```

the error is gone and the app works fine...if you have thoughts as to what might be happening, please do share. 

Thank you XD

**Comment by AndreasFaust** [comment]
I fixed it over the weekend (update to version 1.8.10). Sorry for the confusion.

**Comment by rchrdnsh** [comment]
It looks like it's working just fine again, thank you! XD
